### DISCUSSION TITLE
Arm disappear after pressing animate

### DISCUSSION THREAD

**Discussion by Abdelrahman-Alkhodary** [SofaDefrost/SoftRobots#130] [Help - Building / Configuring] [answered]
Hello everyone

I'm trying to create a soft arm inside SOFA controlled by a cable. The cable goes through the arm and holds to the tip. I have been following the example here 

https://github.com/SofaDefrost/SoftRobots/blob/master/docs/sofapython3/thematicalDocs/T4-DirectActuation/Actuators/Cable/CableForce.py

but I have been facing a problem after I press animate the arm disappears from SOFA without any errors in the SOFA terminal. I ran the example above, but I did not face this problem. I use the last version of SOFA on windows 10.

Here's my code
https://github.com/Abdelrahman-Alkhodary/SoftArm-SOFA

**Comment by EulalieCoevoet** [comment, marked as answer]
Hi,

Thank you for sharing your code it helps a lot.
I think your problem comes from your Poisson's ratio. When you look at the datafield info it says `FEM Poisson Ratio in Hooke's law [0,0.5[`. So you should not set it to `0.5`.

**Reply by Abdelrahman-Alkhodary** [reply]
Thanks for the reply, it worked by changing the Poisson Ratio as you suggested.

**Reply by EulalieCoevoet** [reply]
Great! I think it would be nice to have a warning in such case. I’ll make a PR in SOFA when I have time.

**Reply by Abdelrahman-Alkhodary** [reply]
I think that would be nice as I thought the problem could be in the stl file i'm using in the beginning. Actually, I have two more questions, first what are the units that SOFA use for example in cable controller the value of the force is increased by 1000 and the units of the mechanical objects is it cm or mm. The arm I'm trying to simulate should be actuated by cables with maxForce = 15 Newtons. My other question is in the example when I press the plus key, the finger moves step by step as much as I press the key but when I press the minus key the finger reset to its rest position suddenly not step by step as what happens when you press the plus key. Do you know what the solution could be for this as it also happened with the arm I'm trying to simulate?

**Reply by EulalieCoevoet** [reply]
Sorry I didn't notice this message. For those who would have the same questions I've answered them in this discussion -> https://github.com/SofaDefrost/SoftRobots/discussions/135
